### PR TITLE
🌐 Add Spanish translation of Deployment - Deploy with Docker

### PR DESCRIPTION
**Pull request by pedabraham**
Hello 👋!,
This Pull Request includes the Spanish translation of **deployment/docker.md**

##### This pull request continues with the Spanish translation of #1218

### CODE REVIEW

File: docs/es/docs/deployment/docker.md [hunk 1]
```diff
@@ -1,51 +1,51 @@
...
+# Deploy con Docker
 
-In this section you'll see instructions and links to guides to know how to:
+En esta sección verás instrucciones y enlaces a guías para saber como:
 
-* Make your **FastAPI** application a Docker image/container with maximum performance. In about **5 min**.
-* (Optionally) understand what you, as a developer, need to know about HTTPS.
-* Set up a Docker Swarm mode cluster with automatic HTTPS, even on a simple $5 USD/month server. In about **20 min**.
-* Generate and deploy a full **FastAPI** application, using your Docker Swarm cluster, with HTTPS, etc. In about **10 min**.
+* Hacer de tu app con **FastAPI**, una imagen/contenedor de Docker con un máximo rendimiento. En cerca de **5 min**.
```
Comment (jovicon) [inline docs/es/docs/deployment/docker.md:5]: You can change: `Hacer de` for `Convierte a` or `Haz`

File: docs/es/docs/deployment/docker.md [hunk 2]
```diff
@@ -1,51 +1,51 @@
...
+* (Opcionalmente) entender que, como desarrollador, necesitas conocer sobre HTTPS.
+* Configurar un cluster de Docker Swarm mode con HTTPS automatico, incluso en un simple servidor por $5 USD/mes. En cerca de **20 min**.
+* Generar y desplegar en toda su totalidad una aplicación con **FastAPI**, usando tu cluster de Docker Swarm, con HTTPS, etc. En cerca de **10 min**.
 
-You can use <a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a> for deployment. It has several advantages like security, replicability, development simplicity, etc.
+Usar <a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a> para el deployment, tiene algunas ventajas como seguridad, replicabilidad, simplicdad en el desarrollo, etc.
```
Comment (jovicon) [inline docs/es/docs/deployment/docker.md:10]: * change `Usar` for `Puedes usar`
* `several` translation is `varias` `distintas`
* change `simplicdad` for `simplicidad`

File: docs/es/docs/deployment/docker.md [hunk 3]
```diff
@@ -1,51 +1,51 @@
...
 
-If you are using Docker, you can use the official Docker image:
+Si estás usando Docket, puedes usar la documentación oficial de Docker en:
```
Comment (jovicon) [inline docs/es/docs/deployment/docker.md:12]: * `Docket` by `Docker`
* `you can use the official Docker image` translation is `puedes usar la imagen oficial de Docker:`

File: docs/es/docs/deployment/docker.md [hunk 4]
```diff
@@ -1,51 +1,51 @@
...
 
 ## <a href="https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker" class="external-link" target="_blank">tiangolo/uvicorn-gunicorn-fastapi</a>
 
-This image has an "auto-tuning" mechanism included, so that you can just add your code and get very high performance automatically. And without making sacrifices.
+Esta imagen tiene un mecanismo de "auto-calibración" incluido, así que puedes solamente añadir tu código y obtener muy alto rendimiento de forma automática. Y sin necesidad de hacer sacrificios.
 
-But you can still change and update all the configurations with environment variables or configuration files.
+Pero aun puedes cambiar y actualizar todas las configuraciones con variables del environment o otros archivos de configuración.
```
Comment (jovicon) [inline docs/es/docs/deployment/docker.md:18]: If you will use `environment` without translation you could change `del` for `de`

File: docs/es/docs/deployment/docker.md [hunk 5]
```diff
@@ -1,51 +1,51 @@
...
 
 !!! tip
-    To see all the configurations and options, go to the Docker image page: <a href="https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker" class="external-link" target="_blank">tiangolo/uvicorn-gunicorn-fastapi</a>.
+    Para ver toda la configuración y opciones, ve a la pagina de la imagen de Docker: <a href="https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker" class="external-link" target="_blank">tiangolo/uvicorn-gunicorn-fastapi</a>.
```
Comment (jovicon) [inline docs/es/docs/deployment/docker.md:21]: `configurations` is plural. so, translation must be `todas las configuraciones`

File: docs/es/docs/deployment/docker.md [hunk 6]
```diff
@@ -118,62 +118,62 @@ $ docker run -d --name mycontainer -p 80:80 myimage
...
 
-You will see the alternative automatic documentation (provided by <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>):
+Veras la documentación alternativa automática de tu API (proveída por <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>):
 
 ![ReDoc](https://fastapi.tiangolo.com/img/index/index-02-redoc-simple.png)
 
 ## Traefik
 
-<a href="https://traefik.io/" class="external-link" target="_blank">Traefik</a> is a high performance reverse proxy / load balancer. It can do the "TLS Termination Proxy" job (apart from other features).
+<a href="https://traefik.io/" class="external-link" target="_blank">Traefik</a> es un balanceador de carga/proxy inverso de alto rendimiento. Puede hacer la tarea de "TLS Termination Proxy" (ademas de otras características).
```
Comment (jovicon) [inline docs/es/docs/deployment/docker.md:151]: `job` means `trabajo`

File: docs/es/docs/deployment/docker.md [hunk 7]
```diff
@@ -118,62 +118,62 @@ $ docker run -d --name mycontainer -p 80:80 myimage
...
 ---
 
-With this information and tools, continue with the next section to combine everything.
+Con esta información y herramientas, continua con la siguiente sección para combinar todo.
 
-## Docker Swarm mode cluster with Traefik and HTTPS
+## Docker Swarm mode cluster con Traefik y HTTPS
 
-You can have a Docker Swarm mode cluster set up in minutes (about 20 min) with a main Traefik handling HTTPS (including certificate acquisition and renewal).
+Puedes configurar un cluster de Docker Swarn mode, en minutos (cerca de 20) con un manejador de HTTPS de Traefik.
```
Comment (jovicon) [inline docs/es/docs/deployment/docker.md:163]: change `Swarn` for `Swarm`

File: docs/es/docs/deployment/docker.md [hunk 8]
```diff
@@ -118,62 +118,62 @@ $ docker run -d --name mycontainer -p 80:80 myimage
...
+### <a href="https://medium.com/@tiangolo/docker-swarm-mode-and-traefik-for-a-https-cluster-20328dba6232" class="external-link" target="_blank">Docker Swarm Mode y Traefik para un HTTPS cluster</a>
 
-### Deploy a FastAPI application
+### Deploy una aplicación de FastAPI
 
-The easiest way to set everything up, would be using the [**FastAPI** Project Generators](../project-generation.md){.internal-link target=_blank}.
+La manera mas facil de configurar todo, sera usando los [Project Generators de **FastAPI**](../project-generation.md){.internal-link target=_blank}.
 
-It is designed to be integrated with this Docker Swarm cluster with Traefik and HTTPS described above.
+Esta diseñado para ser integrado con este cluster de Docker Swarm con Traefik y HTTPSS descrito arriba.
```
Comment (jovicon) [inline docs/es/docs/deployment/docker.md:175]: Change `HTTPSS` for `HTTPS`

**Review by jovicon — COMMENTED** [review]
Hi @pedabraham I let you some comments here.

File: docs/es/docs/deployment/docker.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
+# Deploy con Docker
+
+En esta sección verás instrucciones y enlaces a guías para saber como:
+
+* Hacer de tu app con **FastAPI**, una imagen/contenedor de Docker con un máximo rendimiento. En cerca de **5 min**.
```
Comment (pedabraham) [inline docs/es/docs/deployment/docker.md:5]:
```suggestion
* Haz de tu app con **FastAPI**, una imagen/contenedor de Docker con un máximo rendimiento. En cerca de **5 min**.
```

File: docs/es/docs/deployment/docker.md [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+En esta sección verás instrucciones y enlaces a guías para saber como:
+
+* Haz de tu app con **FastAPI**, una imagen/contenedor de Docker con un máximo rendimiento. En cerca de **5 min**.
+* (Opcionalmente) entender que, como desarrollador, necesitas conocer sobre HTTPS.
+* Configurar un cluster de Docker Swarm mode con HTTPS automatico, incluso en un simple servidor por $5 USD/mes. En cerca de **20 min**.
+* Generar y desplegar en toda su totalidad una aplicación con **FastAPI**, usando tu cluster de Docker Swarm, con HTTPS, etc. En cerca de **10 min**.
+
+Usar <a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a> para el deployment, tiene algunas ventajas como seguridad, replicabilidad, simplicdad en el desarrollo, etc.
+
+Si estás usando Docket, puedes usar la documentación oficial de Docker en:
```
Comment (pedabraham) [inline docs/es/docs/deployment/docker.md:12]:
```suggestion
Si estás usando Docker, puedes usar la imagen oficial de Docker en:
```

File: docs/es/docs/deployment/docker.md [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+
+## <a href="https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker" class="external-link" target="_blank">tiangolo/uvicorn-gunicorn-fastapi</a>
+
+Esta imagen tiene un mecanismo de "auto-calibración" incluido, así que puedes solamente añadir tu código y obtener muy alto rendimiento de forma automática. Y sin necesidad de hacer sacrificios.
+
+Pero aun puedes cambiar y actualizar todas las configuraciones con variables del environment o otros archivos de configuración.
```
Comment (pedabraham) [inline docs/es/docs/deployment/docker.md:18]:
```suggestion
Pero aun puedes cambiar y actualizar todas las configuraciones con variables de environment o otros archivos de configuración.
```

File: docs/es/docs/deployment/docker.md [hunk 12]
```diff
@@ -0,0 +1,179 @@
...
+
+!!! tip
+    Para ver toda la configuración y opciones, ve a la pagina de la imagen de Docker: <a href="https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker" class="external-link" target="_blank">tiangolo/uvicorn-gunicorn-fastapi</a>.
```
Comment (pedabraham) [inline docs/es/docs/deployment/docker.md:21]:
```suggestion
    Para ver todas las configuraciones y opciones, ve a la pagina de la imagen de Docker: <a href="https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker" class="external-link" target="_blank">tiangolo/uvicorn-gunicorn-fastapi</a>.
```

File: docs/es/docs/deployment/docker.md [hunk 13]
```diff
@@ -0,0 +1,179 @@
...
+
+Ademas tiene integraciones con Docker. Por lo que, puedes declarar tus dominios en cada configuración de tu aplicación y poder leer esas configuraciones, generar el certificado HTTPS y servir HTTPS a tu aplicación de manera automática, sin requerir algún cambio en la configuración.
+
+---
+
+Con esta información y herramientas, continua con la siguiente sección para combinar todo.
+
+## Docker Swarm mode cluster con Traefik y HTTPS
+
+Puedes configurar un cluster de Docker Swarn mode, en minutos (cerca de 20) con un manejador de HTTPS de Traefik.
```
Comment (pedabraham) [inline docs/es/docs/deployment/docker.md:163]:
```suggestion
Puedes configurar un cluster de Docker Swarm mode, en minutos (cerca de 20) con un manejador de HTTPS de Traefik.
```

File: docs/es/docs/deployment/docker.md [hunk 14]
```diff
@@ -0,0 +1,179 @@
...
+
+Para configurar un cluster de Docker Swarm Mode con Traefik y manejo de HTTPS, sigue esta guía:
+
+### <a href="https://medium.com/@tiangolo/docker-swarm-mode-and-traefik-for-a-https-cluster-20328dba6232" class="external-link" target="_blank">Docker Swarm Mode y Traefik para un HTTPS cluster</a>
+
+### Deploy una aplicación de FastAPI
+
+La manera mas facil de configurar todo, sera usando los [Project Generators de **FastAPI**](../project-generation.md){.internal-link target=_blank}.
+
+Esta diseñado para ser integrado con este cluster de Docker Swarm con Traefik y HTTPSS descrito arriba.
```
Comment (pedabraham) [inline docs/es/docs/deployment/docker.md:175]:
```suggestion
Esta diseñado para ser integrado con este cluster de Docker Swarm con Traefik y HTTPS descrito arriba.
```

File: docs/es/docs/deployment/docker.md (hunk 6 above)
Comment (pedabraham) [reply docs/es/docs/deployment/docker.md:151]: I think that in this case tarea still work, do you think that: 
```suggestion
<a href="https://traefik.io/" class="external-link" target="_blank">Traefik</a> es un balanceador de carga/proxy inverso de alto rendimiento. Puede hacer el trabajo de "TLS Termination Proxy" (ademas de otras características).
```
works better?, 🤔

File: docs/es/docs/deployment/docker.md [hunk 15]
```diff
@@ -0,0 +1,179 @@
+# Deploy con Docker
+
+En esta sección verás instrucciones y enlaces a guías para saber como:
+
+* Haz de tu app con **FastAPI**, una imagen/contenedor de Docker con un máximo rendimiento. En cerca de **5 min**.
+* (Opcionalmente) entender que, como desarrollador, necesitas conocer sobre HTTPS.
+* Configurar un cluster de Docker Swarm mode con HTTPS automatico, incluso en un simple servidor por $5 USD/mes. En cerca de **20 min**.
+* Generar y desplegar en toda su totalidad una aplicación con **FastAPI**, usando tu cluster de Docker Swarm, con HTTPS, etc. En cerca de **10 min**.
+
+Usar <a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a> para el deployment, tiene algunas ventajas como seguridad, replicabilidad, simplicdad en el desarrollo, etc.
```
Comment (pedabraham) [inline docs/es/docs/deployment/docker.md:10]:
```suggestion
Puedes usar <a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a> para el deployment. Tiene distintas ventajas como seguridad, replicabilidad, simplicdad en el desarrollo, etc.
```

File: docs/es/docs/deployment/docker.md [hunk 16]
```diff
@@ -0,0 +1,179 @@
+# Deploy con Docker
+
+En esta sección verás instrucciones y enlaces a guías para saber como:
+
+* Haz de tu app con **FastAPI**, una imagen/contenedor de Docker con un máximo rendimiento. En cerca de **5 min**.
+* (Opcionalmente) entender que, como desarrollador, necesitas conocer sobre HTTPS.
+* Configurar un cluster de Docker Swarm mode con HTTPS automatico, incluso en un simple servidor por $5 USD/mes. En cerca de **20 min**.
+* Generar y desplegar en toda su totalidad una aplicación con **FastAPI**, usando tu cluster de Docker Swarm, con HTTPS, etc. En cerca de **10 min**.
+
+Puedes usar <a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a> para el deployment. Tiene distintas ventajas como seguridad, replicabilidad, simplicdad en el desarrollo, etc.
```
Comment (pedabraham) [inline docs/es/docs/deployment/docker.md:10]:
```suggestion
Puedes usar <a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a> para el deployment. Tiene distintas ventajas como seguridad, replicabilidad, simplicidad en el desarrollo, etc.
```

File: docs/es/docs/deployment/docker.md (hunk 6 above)
Comment (jovicon) [reply docs/es/docs/deployment/docker.md:151]: Yes, 'tarea' sounds very well!

**Review by jovicon — APPROVED** [review]

**Review by bezaca — APPROVED** [review]

**Comment by tiangolo** [comment]
Thanks @pedabraham! 🤓 

As the original in English changed a lot, with a lot more info and different guidelines, I'm gonna pass on this one. But thanks for the effort! ☕

**Pull request closed without merging**